<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="1" name="modelo.modelo_tests.TestFindMethods-20160213112607" tests="3" time="0.113">
	<testcase classname="modelo.modelo_tests.TestFindMethods" name="testsave" time="0.042"/>
	<testcase classname="modelo.modelo_tests.TestFindMethods" name="testfind" time="0.062">
		<failure message="Articulo('n', 'n') != Articulo('n', 'n')" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/home/sebastian/proyectos/SiGeSt/src/unittest/python/modelo/modelo_tests.py", line 30, in testfind
    self.assertEqual(self.artMock.find(self.idArt), result)
AssertionError: Articulo('n', 'n') != Articulo('n', 'n')
]]>		</failure>
	</testcase>
	<testcase classname="modelo.modelo_tests.TestFindMethods" name="testrepr" time="0.009">
		<error message="Instance &lt;Articulo at 0xb5d0c9ac&gt; is not bound to a Session; attribute refresh operation cannot proceed" type="DetachedInstanceError">
<![CDATA[Traceback (most recent call last):
  File "/home/sebastian/proyectos/SiGeSt/src/unittest/python/modelo/modelo_tests.py", line 23, in testrepr
    self.assertEqual(repr(self.artMock),'Articulo(%r, %r)'
  File "/home/sebastian/proyectos/SiGeSt/src/main/python/modelo/modelo.py", line 33, in __repr__
    return 'Articulo(%r, %r)'%(self.id, self.descripcion)
  File "/usr/lib/python3.5/site-packages/sqlalchemy/orm/attributes.py", line 237, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/usr/lib/python3.5/site-packages/sqlalchemy/orm/attributes.py", line 578, in get
    value = state._load_expired(state, passive)
  File "/usr/lib/python3.5/site-packages/sqlalchemy/orm/state.py", line 474, in _load_expired
    self.manager.deferred_scalar_loader(self, toload)
  File "/usr/lib/python3.5/site-packages/sqlalchemy/orm/loading.py", line 610, in load_scalar_attributes
    (state_str(state)))
sqlalchemy.orm.exc.DetachedInstanceError: Instance <Articulo at 0xb5d0c9ac> is not bound to a Session; attribute refresh operation cannot proceed
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
